Detectors/FIT/FV0/simulation/src/Digitizer.cxx

```diff
@@ -100,14 +89,31 @@ void Digitizer::process(const std::vector<o2::fv0::Hit>& hits,
   LOG(INFO) << "[FV0] Digitizer::process(): begin with " << hits.size() << " hits";
 
   auto sorted_hits{hits};
```

Comment:
Here you are cloning the whole vector then doing lot of copying of relatively large Hit objects during the in-place sorting. 
Why don't you sort an index and retrieve the hits from the original array, e.g.
````
std::vector<int> hitIdx(hits.size());
std::iota(hitIdx.begin(),hitIdx.end(),0);
std::sort(hitIdx.begin(),hitIdx.end(), [&](int a,int b) {retun hits[a].GetTrackID() < hits[b].GetTrackID();}  );

// use ordered hits
for (auto ids : hitIdx) {
  const auto& hit = hits[ids];
  ///...
}
````

Reply:
Done.